manifest.yaml

```diff
@@ -0,0 +1,3 @@
+version: 0.6.0
```

Comment:
Although manifests are clearly used for the projects we build RPMs with, I'm not sure what _this_ manifest is up to? Is this more of an example?